src/openfluid/waresdev/WareSrcContainer.cpp

```diff
@@ -550,6 +550,8 @@ void WareSrcContainer::configure()
 {
   if (!openfluid::utils::CMakeProxy::isAvailable())
   {
+    throw openfluid::base::FrameworkException(OPENFLUID_CODE_LOCATION,
+                                              "CMake program not available");
     return;
```

Comment:
The return statement cannot be reached as it is straight after the exception throwing.
It should be removed